Formula/coreutils.rb

```diff
@@ -41,10 +41,16 @@ class Coreutils < Formula
   def install
     system "./bootstrap" if build.head?
 
+    # Fix configure: error: you should not run configure as root
+    on_linux do
+      ENV["FORCE_UNSAFE_CONFIGURE"] = "1" if ENV["HOMEBREW_GITHUB_ACTIONS"]
+    end
```

Comment:
this build just really shouldn't run as root. But that is a problem for another time.

Reply:
Yes. If we find a clean way to handle running docker without root, and without breaking the whole CI, I'm game to remove all the root hacks we have. If anyone wants to start on working on a better solution, go ahead. I'll have time once the linux migration is done.